### PR TITLE
manual improved and remove \n and installer

### PR DESCRIPTION
**Pull request by DPS0340**
:D

### CODE REVIEW

**Comment by DPS0340** [comment]
and removed \n in password. please check.

**Comment by DPS0340** [comment]
i've just made a installer.

**Comment by sourishsarmah** [comment]
@DPS0340 The README updates that you have made are not needed anymore. 
#7 Is in progress, which automates the process of configuring god's eye and saving and encrypting the password.
But the installation step of adding the PATH is required.
I have suggested some changes that you can make.

File: OpenGodsEye.sh [hunk 1]
```diff
@@ -1 +1,6 @@
-chmod +x god
\ No newline at end of file
+#!/bin/sh
+
+chmod +x god
+password=$1
+echo $password > config.ge
```
Comment (sourishsarmah) [inline OpenGodsEye.sh]:
```suggestion
echo $password > config.ge   // not needed

```

File: OpenGodsEye.sh [hunk 2]
```diff
@@ -1 +1,6 @@
...
+PATH=$PATH:$(pwd)
```
Comment (sourishsarmah) [inline OpenGodsEye.sh]: Can you change it so that it adds the path to .bashrc file?

File: README.md [hunk 3]
```diff
@@ -10,17 +10,14 @@ This is a command line tool that substitute the ` sudo ` command so that we dono
...
 
 
-##  Configure God's Eye
-
-1. Create a file name ` config.ge ` and write your sudo password  
-
-2. Add the directory to the PATH
+3. Run the script :  
 
+        bash ./OpenGodsEye.sh (your_sudo_password)
```
Comment (sourishsarmah) [inline README.md]: No need to ask for password

File: OpenGodsEye.sh [hunk 4]
```diff
@@ -1 +1,6 @@
-chmod +x god
\ No newline at end of file
+#!/bin/sh
+
+chmod +x god
+password=$1
```
Comment (sourishsarmah) [inline OpenGodsEye.sh]:
```suggestion

```

**Comment by DPS0340** [comment]
Thanks for reply. let me think.

**Comment by DPS0340** [comment]
not yet. need fix.

**Comment by DPS0340** [comment]
screenshot_20181224_230742
now it works properly on my computer.

**Pull request merged**